lib/utils/socketChainHelper.js

```diff
@@ -44,19 +48,19 @@ const processServerResponse = toString => function processServerResponseHandler(
 		}
 	}
 	// eval success
-	if (res.result === 'success' && res.contentType === 'eval') {
+	if (isSuccess(res.result) && res.contentType === 'eval') {
```

Comment:
all nice for me, but i think that `isSuccess` is redundant:
as `isSuccess` used only inside `processServerResponseHandler` we can drop this function and create variable instead:
`const isSuccess = res.result === 'success' || res.result === 'warning';`
als we can create the similar const for eval line:
`const isEval = res.contentType === 'eval'` e.t.c